r/R/marker_heatmap.R

```diff
@@ -14,5 +14,15 @@ runMarkerHeatmap <- function(req, data) {
   top_markers <- getTopMarkerGenes(nFeatures, data, cellSets)
   top_markers <- getMarkerNames(data, top_markers)
 
-  return(getExpressionValues(top_markers, data))
+  res <- getExpressionValues(top_markers, data)
+  if (!length(res$rawExpression)) {
+    stop(
+      generateErrorMessage(
+        error_codes$NO_MARKER_GENES,
+        "Couldn't get marker genes"
+      )
+    )
+  }
```

Comment:
@ogibson-biomage do we need this? This was a check that was moved from the python code to R code but I can't imagine a scenario where we actually get here

Reply:
I've never seen this stop actually executed. Fell free to remove this check

Reply:
removed